x/docs/internal/keeper/test_commons.go

```diff
@@ -1,7 +1,7 @@
 package keeper
 
 import (
-	"github.com/commercionetwork/commercionetwork/types"
+	types2 "github.com/commercionetwork/commercionetwork/x/docs/internal/types"
```

Comment:
I think we can remove this renaming

```suggestion
	"github.com/commercionetwork/commercionetwork/x/docs/internal/types"
```